README.md

```diff
@@ -462,7 +462,7 @@ There're also some great commercial libraries, like [amchart](https://www.amchar
 
 * [store.js](https://github.com/marcuswestin/store.js) - LocalStorage wrapper for all browsers without using cookies or flash. Uses localStorage, globalStorage, and userData behavior under the hood.
 * [localForage](https://github.com/mozilla/localForage) - Offline storage, improved. Wraps IndexedDB, WebSQL, or localStorage using a simple but powerful API.
-* [jStorage](https://github.com/andris9/jStorage) - jStorage is a simple key/value database to store data on browser side.
+* [jStorage](https://github.com/andris9/jStorage) - jStorage is a simple key/value database to store data on the browser side.
```

Comment:
It's their own description, but the project is in a frozen status. I'm starting to think that we need another column or something to indicate the project status: active, frozen, archived, gone for good, something else.